src/SharedElements.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react';
+import { css, cx } from 'emotion';
+import { textSans, headline, body } from '@guardian/src-foundations/typography';
+import { news, neutral } from '@guardian/src-foundations/palette';
+import { SvgMinus, SvgPlus } from '@guardian/src-icons';
+
+/*
+    Elements that are used on multiple atoms (such as the feedback buttons) 
+    can be imported from here
+*/
+
+/// SUMMARY ELEMENT
+
+const showHideStyling = css`
+    background: ${neutral[7]};
+    color: ${neutral[100]};
+    height: 2rem;
+    font-size: 13px;
+    position: absolute;
+    bottom: 0;
+    transform: translate(0, 50%);
+    padding: 0 15px 0 7px;
+    border-radius: 100em;
+    cursor: pointer;
+    display: inline-flex;
+    align-items: center;
+    border: 0;
+    margin: 0;
+    :hover {
+        background: ${news[400]};
+    }
+`;
+
+const plusStyling = css`
+    margin-right: 12px;
+    margin-bottom: 6px;
+    width: 33px;
+    fill: white;
+    height: 28px;
+`;
+
+const minusStyling = css`
+    margin-right: 14px;
+    margin-bottom: 6px;
+    width: 30px;
+    fill: white;
+    height: 25px;
+    padding-left: 4px;
+`;
+
+const iconSpacing = css`
+    display: inline-flex;
+    align-items: center;
+    ${textSans.small()};
+`;
+
+export const Summary = ({
+    sectionTitle,
+    title,
+    expandHandler,
+}: {
+    sectionTitle: string;
+    title: string;
+    expandHandler: () => void;
+}): JSX.Element => {
+    const [hasBeenExpanded, setHasBeenExpanded] = useState(false);
+    const [expandEventSent, setExpandEventFired] = useState(false);
+    return (
+        <summary
+            onClick={() => {
+                if (!expandEventSent) {
+                    expandHandler();
+                    setExpandEventFired(true);
+                }
+                setHasBeenExpanded(!hasBeenExpanded);
+            }}
+        >
+            <span
+                className={css`
+                    color: ${news[400]};
+                    display: block;
+                    ${body.medium({
+                        lineHeight: 'tight',
+                        fontWeight: 'bold',
+                    })};
+                `}
+            >
+                {sectionTitle}
+            </span>
+            <h4
+                className={css`
```

Comment:
Maybe don't inline this css? It's nice to give them a name for readability here and to be able to reference back when they're rendered in the browser.

Reply:
(or the one just above)